scripts/generate-upgrade-dashboard.js

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import semver from 'semver';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// get latest stable Backstage release
+async function getLatestBackstageVersion() {
+  const response = await fetch(
+    'https://api.github.com/repos/backstage/backstage/releases',
```

Comment:
Could we use this instead: https://versions.backstage.io/v1/tags/main/manifest.json. Might make things simpler?

Reply:
Changed in 7fb9530d0da0d55d31389c80fea22b8e50e6e98e